modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,42 +7,19 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+For Podman 5.x, the `/home/user/.config` must be owned by the current user for Podman to function correctly.
+The `storage.conf` file for Podman is typically stored in this folder.
+This document explains how to configure the workspace's container entrypoint script for the workspace so that fuse-overlayfs is being used for all workspaces using that container.
+The Universal Developer Image (UDI) already contains the necessary configuration by default.
```

Comment:
```suggestion
Learn about configuring the workspace's container entrypoint script so that fuse-overlayfs is used for all workspaces using that container.

The Universal Developer Image (UDI) already contains the necessary configuration by default. However, you must configure the script manually if you use custom images.
```